zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala

```diff
@@ -35,6 +35,7 @@ import scala.collection.JavaConverters._
 
 trait CassandraSpanStoreFactory {self: App =>
 
+  val ensureSchema          = flag[Boolean]  ("zipkin.store.cassandra.schema.ensureExists", false, "ensures schema exists")
```

Comment:
rename this flag to be same depth as others.. `zipkin.store.cassandra.ensureSchema`